github/ci/prow/files/jobs/kubevirt/user-guide/user-guide-postsubmits.yaml

```diff
@@ -31,10 +27,15 @@ postsubmits:
           git config --global user.email "$GIT_AUTHOR_EMAIL"
           git config --global user.name "$GIT_AUTHOR_NAME"
           commit=$(git show -s --format=%H)
+          (cd .. && git clone --branch gh-pages "https://kubevirt-bot@github.com/kubevirt/user-guide.git" user-guide-gh-pages)
+          USER_GUIDE_WORK_DIR=$(cd ../user-guide-gh-pages && pwd)
           make build
-          cd ../user-guide-gh-pages
+          # this check is required as the directory might not exist if anything goes wrong!
```

Comment:
My only doubt is if this is necessary... you talk about handling this during `make build` on a pre-submit job, but wouldn't it be enough/better to rely on the exit status of the `make build` invocation just above here?

Reply:
Yes, sure. But apparently the exit code is **not** non zero at the moment in case anything goes wrong in the `make build`. In the situation I observed the `site` folder wasn't even present while the exit code was zero.

But you are right, if the site folder is non existing the script will fail on it's own. I'm going to remove the check.

Reply:
In fact I actually didn't realize that the script failed because the make target didn't even create the site folder. So I thought it would be better to explicitly check for existence so that it would be more clear why it failed.

Reply:
I had a bad echo lingering at the end of `make build` causing exit non 0 to become exit 0.  I greatly simplified the build script.  We now fully rely on Makefile's ability to handle exit codes rather than have the rule be a continuous bash script and try to capture exit codes. see [f68c6584b](https://github.com/kubevirt/user-guide/pull/372/commits/f68c6584baea5ffb2a14a601af6ae65f8a1b1499)

I really don't care what the script looks like before/after `make build`.  @codificat , let's not get stuck in the mud here.  We need to get moving. @dhiller if this works then lgtm/approve.

Reply:
I've removed the check now, I've tested it, and works.